libmodernize/fixes/fix_itertools_imports_six.py

```diff
@@ -39,10 +42,15 @@ def transform(self, node, results):
                 assert child.type == syms.import_as_name
                 name_node = child.children[0]
             member_name = name_node.value
-            if member_name in ('imap', 'izip', 'ifilter',
-                               'ifilterfalse', 'izip_longest'):
+            if member_name in (
+                "imap",
+                "izip",
+                "ifilter",
+                "ifilterfalse",
+                "izip_longest",
+            ):
                 child.value = None
-                libmodernize.touch_import(u'six.moves', member_name[1:], node)
+                libmodernize.touch_import("six.moves", member_name[1:], node)
```

Comment:
Was this on purpose? I'm not sure if it really has impact, but it is a change in type under Python 2.

It happens in a couple of places.

Reply:
modernize is 3.6+ only because fissix is 3.6 only. Also the file starts with `generator_stop` which implies `unicode_literals`